### PR TITLE
pytest special test case 3

### PR DESCRIPTION
**Pull request by PRATHAM-SPS**
### Details:
 - *Replaced the generator's functionality with pytest*

### Tickets:
 - *110705*

### CODE REVIEW

**Comment by PRATHAM-SPS** [comment]
Hello @rkazants  When I test this file with unittest  12 test cases get failed...
So what I have done is coverted it into pytest  keeping the number of failed test cases same as unittest.

**Comment by p-wysocki** [comment]
@rkazants could you please take a look?

**Comment by p-wysocki** [comment]
build_jenkins

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
Hi @PRATHAM-SPS,

Please check the test, now there is a failure:

```
install/tests/mo/unit_tests/mo/utils/freeze_placeholder_test.py:165: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/__t/Python/3.11.6/x64/lib/python3.11/site-packages/openvino/tools/mo/convert_impl.py:409: in prepare_ir
    ngraph_function = moc_pipeline(argv, moc_front_end)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

argv = Namespace(feManager=<FrontEndManager>, extensions=['/__w/openvino/openvino'], use_legacy_frontend=None, use_new_fronte...2'>, 'in2': <class 'numpy.float32'>}, unnamed_freeze_placeholder_with_value={}, layout_values=[], mean_scale_values={})
moc_front_end = <FrontEnd 'onnx'>

    def moc_pipeline(argv: argparse.Namespace, moc_front_end: FrontEnd):
        """
        Load input model and convert it to nGraph function
        :param: argv: parsed command line arguments
        :param: moc_front_end: Loaded Frontend for converting input model
        :return: converted nGraph function ready for serialization
        """
        input_checkpoint = getattr(argv, 'input_checkpoint', None)
        share_weights = getattr(argv, 'share_weights', True)
        if argv.input_model and input_checkpoint:
            # frozen format with v1 checkpoints
            input_model = moc_front_end.load([argv.input_model, argv.input_checkpoint], share_weights)
        elif argv.input_model:
>           input_model = moc_front_end.load(argv.input_model, share_weights)
E           RuntimeError: Exception from src/frontends/onnx/onnx_common/src/parser.cpp:21:
E           Could not open the file: test_model.onnx
```

File: tools/mo/unit_tests/mo/utils/freeze_placeholder_test.py [hunk 1]
```diff
@@ -61,8 +60,7 @@ def get_test_default_frontends():
     return {"onnx": "new", "tf": "legacy"}
 
 
-@generator
-class TestMoFreezePlaceholder(unittest.TestCase):
+class TestMoFreezePlaceholder():
     def setUp(self):
```
Comment (rkazants) [inline tools/mo/unit_tests/mo/utils/freeze_placeholder_test.py:64]: probably `setUp` is not longer working now that is why the input model is not found. Please fix

**Comment by PRATHAM-SPS** [comment]
> build_jenkins

@rkazants  can you please merge this PR, and please simply merge the PR don't squeeze and merge.

Thankyou,
Pratham Ingawale

**Comment by rkazants** [comment]
@PRATHAM-SPS, merged. We always squash and merge PRs. Sorry.
@PRATHAM-SPS, thanks a lot for your contribution. Can you please proceed with removing test-generator dependencies in other parts. If you are completely done, can you please try to remove it from requirements.txt files in OV?

Best regards,
Roman

**Comment by PRATHAM-SPS** [comment]
> @PRATHAM-SPS, merged. We always squash and merge PRs. Sorry. @PRATHAM-SPS, thanks a lot for your contribution. Can you please proceed with removing test-generator dependencies in other parts. If you are completely done, can you please try to remove it from requirements.txt files in OV?
> 
> Best regards, Roman

yes, some files are remain i will create PR for them , squash and merge PR reflacts only one dot on profile :(

**Pull request merged**